src/control/security/pem.go

```diff
@@ -38,33 +38,43 @@ import (
 )
 
 const (
-	SafeKeyPerm  os.FileMode = 0400
-	SafeCertPerm os.FileMode = 0664
-	SafeDirPerm  os.FileMode = 0700
+	MaxKeyPerm  os.FileMode = 0700
```

Comment:
Wouldn't it be better to have 0600? Executing the key doesn't make much sense to me.

Reply:
Ideally 0600 would be better but there is no harm in it being 0700 and that is what openssh caps the permissions at as well.

Reply:
Works for me.